src/app/components/PortraitVideoCarousel/index.tsx

```diff
@@ -0,0 +1,131 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext, useEffect, useRef, useState } from 'react';
+import Heading from '../Heading';
+import { LeftChevron, RightChevron } from '../icons';
+import styles, { PROMO_ITEM_WIDTH } from './index.styles';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+interface PortraitVideoItem {
+  id: string;
+  images?: { url: string; altText?: string }[];
+  headlines?: { promoHeadline?: string };
+  link?: { path: string };
+}
+
+interface PortraitVideoCarouselProps {
+  title: string;
+  items: PortraitVideoItem[];
+}
+
+const PortraitVideoCarousel = ({
+  title,
+  items,
+}: PortraitVideoCarouselProps) => {
+  const { dir } = useContext(ServiceContext);
+  const scrollRef = useRef<HTMLDivElement>(null);
+  const [canScrollLeft, setCanScrollLeft] = useState(false);
```

Comment:
I'm that Simorgh is largely stateless but I couldn't think of an alternative to allow the css for the navigation to track the end points left and right of the carousel behaviour so have used `state` hooks here for the scroll. Perhaps it ca be simplified further for production